en/Obsidian Sync/Select files and settings to sync.md

```diff
@@ -29,34 +47,54 @@ Common examples of hidden folders that will not be synced include `.vscode`, `
+The primary device is considered your source-of-truth. This is the device on which you make changes that you want to sync across all other devices.
 
 1. Access **Settings** → **Sync**.
 2. Activate the desired settings under **Vault configuration sync**.
-3. Restart Obsidian. On mobile or tablet, this may require a force quit.
+3. Reload or Restart Obsidian. On mobile or tablet, this may require a force quit.
 4. Allow time for the settings to synchronize with your remote vault.
 
 ### Secondary device(s)
 
-The secondary device is another device you use Obsidian with (e.g. your phone). It can also mean the devices you wish to receive an updated change on.
+The secondary device is any other device you use with Obsidian, such as your phone. It refers to devices that receive updates from the primary device.
 
 1. Navigate to **Settings** → **Sync**.
 2. Enable the desired settings under **Vault configuration sync**.
 3. Wait for the changes to be downloaded from your remote vault.
-4. Restart the application to ensure that your synchronized settings take effect. On mobile or tablet, this may require a force quit.
+4. Reload or Restart the application to ensure that your synchronized settings take effect. On mobile or tablet, this may require a force quit.
+
+## Hot reloading of settings
+
+> [!question] Wondering what Hot Reloading is? Hot reloading refreshes only the modified files without resetting the app's state, but it does not necessarily apply those changes without a reload.
+
+Obsidian has limited ability to update and apply configurations received from Sync. The following configurations will hot reload:
+
+- Most Obsidian app configurations, including hotkeys and properties.
+- Appearance configurations.
+- Configurations for already enabled Obsidian Core plugins (e.g., the location of the daily notes template folder).
+
+However, for CSS changes involving [[CSS snippets]] or [[Themes]], while the appearance configurations are received, the application must be reloaded to reflect visual changes.
+
+The following configurations will not hot reload, and will require an application reload or restart:
+
+- Graph view configurations.
+- States of Obsidian Core plugins (e.g., whether Daily Notes is enabled).
+- Community plugin configurations.
+
+Additionally, in rare cases, configurations that do not support hot reloading may be merged incorrectly or overwritten completely. For more details, see [[Troubleshoot Obsidian Sync#Conflict resolution|conflict resolution]].
+
+> [!abstract] Community Plugins and Hot-Reloading
+> Community plugin developers can enable hot reloading for their configuration files using the [Plugin.onExternalSettingsChange() method](https://docs.obsidian.md/Reference/TypeScript+API/Plugin/onExternalSettingsChange). However, hot reloading may not be suitable for all plugins and could be disadvantageous in some scenarios.
```

Comment:
I don't think this is an appropriate place to include details about the API. It isn't the right audience and the answer feels incomplete.

I think this information should be on the developer docs site, and if we want to link there, we should just say `"Are you a developer, learn how to [seamlessly integrate your plugin with Sync.](<link>)"`.

Reply:
I did want to call attention to it slightly, to get it more known thats it is a thing. The one liner you suggested better, I will commit an update.